### ISSUE TITLE
Create a GenBank reader

### ISSUE THREAD

**Issue by vmesel** [vmesel/FastaDB#5]
### Description

Most labs already work with GenBank format file like this one(http://wp-a.co/u/6RcB), we could also make an implementation to use this file format to grab informations.

### What should be done

Find a module that supports handling GenBank format, and scrape it to our models, them we should have for information about the gene and whatever was done with it

**Comment by inaciomdrs** [comment]
Yesterday I've started implementating this functionality in a local branch (which is already available at the fork I'm responsible). I just need now to make some tests.

**Comment by vmesel** [comment]
@inaciomdrs we need to see where all data is going to be put on! We have only some fields!

**Comment by inaciomdrs** [comment]
@vmesel Yes, it's true. Idea is passing information to dict fields defined in #2 as it was done with fasta (inclusive, code is very similar). About this, I see that some containing information in genbank file will not be put in fdb file (once there's no field regarding that), and maybe some fields of fdb file be empty because may there be no "correspodent field" in genbank file. This same issue can also be applied to fasta files, it's a stuff to be verified.

**Comment by vmesel** [comment]
Before you do this @inaciomdrs take a look at the bug related on #7 ! It's important to solve it, so it can be exported for any platform, even python!

**Comment by inaciomdrs** [comment]
@vmesel Roger that.
